mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
For you information you will find your pk3dir NetRadiant commits there (working branch):
https://gitlab.com/illwieckz/netradiant/commits/pk3dir
and your GtkRadiant ones there:
https://github.com/illwieckz/GtkRadiant/tree/unvanquished
badly, for this one, it builds but does not browse pk3dirs, I haven't figured out why yet. The pk3dir support you wrote in NetRadiant is far more advanced and featureful though.

All that works is very useful, thanks to have done it!